pkg/config/instrumentation/remote_config_metrics.go

```diff
@@ -34,10 +40,12 @@ func newRemoteConfigMetrics() *remoteConfigMetrics {
 	return &remoteConfigMetrics
 }
 
-func (r *remoteConfigMetrics) InstrumentRemoteConfigFetch(code int) {
-	r.fetchStatusCodes.WithLabelValues(fmt.Sprintf("%d", code)).Inc()
+func InstrumentRemoteConfigFetch(code int) {
```

Comment:
nit: could we rename this param to `statusCode`? Same for label

Reply:
Makes sense, and done!